src/components/createbook.jsx

```diff
@@ -15,25 +16,27 @@ const Create = () => {
     changeInput({ ...input, [e.target.name]: e.target.value });
   };
   const handleSubmit = () => {
-    const newObj = { ...input, item_id: `${Math.random()}` };
-    dispatch(postToAPI(newObj));
-    dispatch(addBooks(newObj));
-    changeInput({
-      title: '',
-      author: '',
-      category: '',
-      item_id: '',
-    });
+    if (input.title && input.category && input.author) {
+      const newObj = { ...input, item_id: `${Math.random()}` };
+      dispatch(postToAPI(newObj));
+      dispatch(addBooks(newObj));
+    } else {
+      showErr('Please add correct parameters');
+      setTimeout(() => {
+        showErr('');
+      }, 2000);
+    }
   };
   return (
     <section className="createsection">
-      <h3>ADD NEW BOOK</h3>
+      <h3 className="addheader">ADD NEW BOOK</h3>
+      <p className="err">{err}</p>
       <form action="" onSubmit={handleSubmit}>
         <ul className="inputcontainer">
           <li className="titlecontainer"><input placeholder="Book title" name="title" onChange={handleChange} defaultValue={input.title} className="title" type="text" /></li>
           <li className="categorycontainer"><input placeholder="Category" name="category" onChange={handleChange} defaultValue={input.category} className="category" type="text" /></li>
           <li className="authorContainer"><input type="text" name="author" onChange={handleChange} placeholder="Author" defaultValue={input.author} className="author" /></li>
-          <li><input type="button" onClick={handleSubmit} value="ADD BOOK" /></li>
+          <li><input className="addbtn" type="button" onClick={handleSubmit} value="ADD BOOK" /></li>
         </ul>
       </form>
     </section>
```

Comment:
Template 
![image](https://user-images.githubusercontent.com/78251927/230718255-529763ac-769c-45de-8504-7c25978e7c9f.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/230718311-a489bead-c25a-4270-bcd5-b56f0fcabd66.png)

Kindly add a background color to the container of the `Book List` as provided on the given template. 👍 

Please try to style all the components as much as possible similar to the given [Zeplin](https://zeplin.io/) design as it is one of the project's core requirements.

![image](https://user-images.githubusercontent.com/78251927/230718608-cdaedbd4-7364-4136-9cba-bf9c48a9fe68.png)

Let's follow the project core requirements for professional projects, and if we practice from our learning phase, it will be easier in the future at the industry level 😎💪. Here is an article on [The importance of understanding software requirements](https://www.freecodecamp.org/news/why-understanding-software-requirements-matter-to-you-as-a-software-engineer/)